x/proof/keeper/msg_server_submit_proof.go

```diff
@@ -165,39 +165,39 @@ func (k msgServer) SubmitProof(ctx context.Context, msg *types.MsgSubmitProof) (
 	if err := relayReq.ValidateBasic(); err != nil {
 		return nil, status.Error(codes.FailedPrecondition, err.Error())
 	}
-	logger.Info("successfully validated relay request")
+	logger.Debug("successfully validated relay request")
```

Comment:
I :100:% agree that these definitely SHOULD be debug logs instead of info logs; however, until we do something to improve the on-chain logging situation, debug logs are pretty useless. The debug logs contain a bunch of cosmos-sdk internal output that buries out logs. 

See: https://github.com/pokt-network/poktroll/blob/main/x/proof/keeper/msg_server_submit_proof.go#L3-L5

Reply:
I left the key logs (see the bottom) as Info. As someone who spent time looking into it, it was getting in the way.

Reply:
@bryanchriswhite If you still disagree.

Reply:
Sounds like you've got more perspective, I trust your opinion. :+1: